docs/src/app/components/pages/components/TimePicker/ExampleComplex.js

```diff
@@ -31,6 +35,13 @@ export default class TimePickerExampleComplex extends React.Component {
           value={this.state.value24}
           onChange={this.handleChangeTimePicker24}
         />
+        <TimePicker
+          format="24hr"
+          hintText="24hr Format with auto ok"
+          autoOk={true}
```

Comment:
I'm not sure the `autoOk` example best location is inside the controlled examples. What about into the simple one?